docs/source/download.rst

```diff
@@ -1,64 +1,103 @@
-   git clone https://github.com/devitocodes/devito.git
-   cd devito
+   # NOTE (admin-sudo access may be needed) for some steps
+
+   # 1. Pull Devito image
+   docker pull devitocodes/devito:cpu-latest
 
-   # 1. run the tests
-   docker-compose run devito /tests
+   # 2. (Optional but recommended) Run the tests
+   docker run --rm --name testrun 'devitocodes/devito:cpu-latest' pytest tests/test_operator.py
 
-   # 2. start a jupyter notebook server on port 8888
-   docker-compose up devito
+   # 3. Start a Jupyter notebook server on port 8888
+   docker run --rm -it -p 8888:8888 devitocodes/devito:cpu-latest
 
-   # 3. start a bash shell with devito
-   docker-compose run devito /bin/bash
+   # 4. Start a bash shell with Devito
+   docker run --rm -it devitocodes/devito:cpu-latest /bin/bash
 
-The three sample commands above have only been included as illustrations of typical
+These sample commands above have only been included as illustrations of typical
 uses of Devito inside a container. They are not required to be run in that order.
+Please also review these commands within the dockerfile itself:
+`Dockerfile <https://github.com/devitocodes/devito/blob/master/docker/Dockerfile>`_
+
+
+Docker for GPUs
+```````````````
+
+.. code-block:: shell
+
+   # Requirements: Install `Nvidia container toolkit <https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html#setting-up-nvidia-container-toolkit>`_
+
+   # NOTE (admin-sudo access may be needed) for some steps
+   # 1. Pull Devito image
+   docker pull devitocodes/devito:gpu-latest
+
+   # 2. (Optional but recommended) Run the tests
+   docker run --gpus all --rm --name testrun 'devitocodes/devito:gpu-latest' pytest tests/test_gpu_openacc.py
+
+   # 3. Start a Jupyter notebook server on port 8888
+   # Not recommended with the GPU docker, please use the `Docker for CPUs` if you aim for the tutorials
+   docker run --gpus all --rm -it -p 8888:8888 devitocodes/devito:gpu-latest
+
+   # 4. Start a bash shell with Devito
+   docker run --gpus all --rm -it devitocodes/devito:gpu-latest /bin/bash
 
-1. Command 1 above runs the unit tests included with Devito to check whether the
-installation succeeded. This is not necessary but a handy first thing to run. As
-long as the first few tests are passing, it is possible to press `Ctrl+C` to stop
+These sample commands above have only been included as illustrations of typical
+uses of Devito inside a container. They are not required to be run in that order.
+Please also review these commands within the dockerfile itself: `Dockerfile <https://github.com/devitocodes/devito/blob/master/docker/Dockerfile>`_
+
+
+| 1. Command 1 pulls the docker image of Devito in your machine.
+
+2. Command 2 above runs the unit tests included with Devito to check whether the 
+installation succeeded. Testing is not necessary but a handy first thing to try running. As
+long as the first few tests pass, it is possible to press `Ctrl+C` to stop
```

Comment:
I would drop this entire sentence about CTRL+C etc

Reply:
definitely agree